students/zandra_eng/finalproject/web.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python 3
+
+"""
+Reference:
+csv file
+https://docs.python.org/2/library/csv.html
+
+matplotlib
+https://matplotlib.org/gallery/pyplots/pyplot_formatstr.html#sphx-glr-gallery-pyplots-pyplot-formatstr-py
+
+
+"""
+
+
+#import datetime
+import sys
+from dateutil.parser import parse
+import matplotlib.pyplot as plt
+import csv
+
+# if pulling data from the internet for python 2 use: import pandas.io.data as web
+
+from pandas_datareader import data as web
+
+
+class StockPlotter:
+
+    def fetch_stock(self, stock_name, start, end):
+        df = web.DataReader(stock_name, "google", start, end)
+        return df
```

Comment:
save these parameters in the instance (self):

```
self.name = stock_name
self.start = start # you maybe don't need these, they are in the data, yes?
self.end = end 
self.df = web.DataReader(stock_name, "google", start, end)
```
now you have them for the other methods to work with.